### PR TITLE
Add duplicate_mediafile view and tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: tests/test_duplicate_mediafile.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    conn = psycopg2.connect(
+        host="media-postgresql",
+        port=5432,
+        database="openslides",
+        user="openslides",
+        password="openslides",
+    )
+    with conn:
+        with conn.cursor() as cur:
+            cur.execute("DELETE FROM mediafile_data WHERE id NOT IN (2, 3)")
```
Comment (GabrielInTheWorld) [inline tests/test_duplicate_mediafile.py:21]: Will this be often used in other places, too? Then extract this into an own function, too.

File: tests/test_duplicate_mediafile.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        "target_id": 5,
+    }
+    response = requests.post(DUPLICATE_URL, json=payload)
+    assert response.status_code == 200
+    assert response.text == ""
+
+    get_response = requests.get(GET_URL + "5")
+    assert get_response.status_code == 200
+    assert get_response.content == b"a2"
+    assert "text/plain" in get_response.headers.get("Content-Type")
```
Comment (GabrielInTheWorld) [inline tests/test_duplicate_mediafile.py:36]: Such asserts could be in an own function, maybe with a variable `content`-parameter (or anything else).

File: tests/test_duplicate_mediafile.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+
+def test_broken_source():
+    payload = {
+        "source_id": 4,
+        "target_id": 5,
+    }
+    response = requests.post(DUPLICATE_URL, json=payload)
+    assert response.status_code == 500
+    assert "message" in response.json()
```
Comment (GabrielInTheWorld) [inline tests/test_duplicate_mediafile.py:46]: See above

File: tests/test_duplicate_mediafile.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+
+def test_broken_target():
+    payload = {
+        "source_id": 2,
+        "target_id": 3,
+    }
+    response = requests.post(DUPLICATE_URL, json=payload)
+    assert response.status_code == 500
+    assert "message" in response.json()
```
Comment (GabrielInTheWorld) [inline tests/test_duplicate_mediafile.py:56]: See above

File: tests/test_duplicate_mediafile.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+
+def test_empty():
+    payload = {}
+    response = requests.post(DUPLICATE_URL, json=payload)
+    assert response.status_code == 400
+    assert "message" in response.json()
```
Comment (GabrielInTheWorld) [inline tests/test_duplicate_mediafile.py:63]: Same as above. You could make the `status_code` a variable parameter in an own function, too.

**Review by GabrielInTheWorld — CHANGES_REQUESTED** [review]

File: src/mediaserver.py [hunk 6]
```diff
@@ -104,6 +104,29 @@ def file_post(file_type):
     return "", 200
 
 
+@app.route("/internal/media/duplicate_mediafile/", methods=["POST"])
+def duplicate_mediafile():
+    try:
+        decoded = request.data.decode()
+        dejson = json.loads(decoded)
+    except Exception:
+        raise BadRequestError("request.data is not json")
```
[the remark this answers is not in the document]
Comment (reiterl) [reply src/mediaserver.py:113]: Use a function get_json_from_request for this.

File: src/mediaserver.py [hunk 7]
```diff
@@ -104,6 +104,29 @@ def file_post(file_type):
...
+    try:
+        source_id = int(dejson["source_id"])
+        target_id = int(dejson["target_id"])
+    except Exception:
+        raise BadRequestError(
+            f"The post request.data is not in right format: {request.data}"
+        )
```
[the remark this answers is not in the document]
Comment (reiterl) [reply src/mediaserver.py:120]: Done.

File: tests/test_duplicate_mediafile.py [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+@pytest.fixture(autouse=True)
+def reset_db_2():
+    """Deletes all mediafiles except for id=2 and id=3 (example data)"""
+    conn = psycopg2.connect(
+        host="media-postgresql",
+        port=5432,
+        database="openslides",
+        user="openslides",
+        password="openslides",
+    )
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/test_duplicate_mediafile.py:18]: Extracted into own function.

File: tests/test_duplicate_mediafile.py (hunk 1 above)
Comment (reiterl) [reply tests/test_duplicate_mediafile.py:21]: Extracted into reset_db function.

File: tests/test_duplicate_mediafile.py (hunk 3 above)
Comment (reiterl) [reply tests/test_duplicate_mediafile.py:46]: Extract check_response.

File: tests/test_duplicate_mediafile.py (hunk 4 above)
Comment (reiterl) [reply tests/test_duplicate_mediafile.py:56]: Extract check_response.

File: tests/test_duplicate_mediafile.py (hunk 5 above)
Comment (reiterl) [reply tests/test_duplicate_mediafile.py:63]: Extract check_response.

File: tests/test_duplicate_mediafile.py (hunk 2 above)
Comment (reiterl) [reply tests/test_duplicate_mediafile.py]: Extracted as function.

**Review by GabrielInTheWorld — APPROVED** [review]

**Pull request merged**